### ISSUE TITLE
Unicode unquoted strings produce script error in windows

### ISSUE THREAD

**Issue by woble** [stylelint/stylelint#1198]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I am not really sure whether this is a script bug or user error (lack of knowledge of unicode strings) but the following CSS and rule halt the script.

> Which rule, if any, is this issue related to?

Seems to be related to `font-family-name-quotes`. Might affect some others as well.

> What CSS is needed to reproduce this issue?

```css
.foo {
    font-family: Hiragino Sans GB, 冬青黑体简体中文, sans-serif;
}
```

> What stylelint configuration is needed to reproduce this issue?

Enclosing `冬青黑体简体中文` in single quotes solves the problem, but as I said, it could be that unicode strings should always be enclosed in quotes.

When using only a single unicode character without quotes, e.g. `冬`, the script works as expected, i.e. Error: Expected single quotes around font-family name `冬`.

Removing the rule does not produce a script error, as expected, since the property and values are not evaluated.

```json
{
  "rules": {
    "font-family-name-quotes": "single-where-required",
  }
}
```

> Which version of stylelint are you using?

e.g. `6.2.2`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> What did you expect to happen?

Linting.

> What actually happened (e.g. what warnings or errors you are getting)?

Script error.

```shell
Error: Subject parameter value width cannot be greater than the container width.
    at exports.default (C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\node_modules\table\dist\alignString.js:100:15)
    at C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\node_modules\table\dist\alignTableData.js:37:50
    at arrayMap (C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\node_modules\lodash\_arrayMap.js:16:21)
    at map (C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\node_modules\lodash\map.js:51:10)
    at C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\node_modules\table\dist\alignTableData.js:29:34
    at arrayMap (C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\node_modules\lodash\_arrayMap.js:16:21)
    at map (C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\node_modules\lodash\map.js:51:10)
    at exports.default (C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\node_modules\table\dist\alignTableData.js:28:30)
    at exports.default (C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\node_modules\table\dist\table.js:131:39)
    at formatter (C:\Users\USERNAME\AppData\Roaming\npm\node_modules\stylelint\dist\formatters\stringFormatter.js:148:33)
```

**Comment by jeddy3** [comment]
@woble Thanks again for another detailed report.

This looks to be an issue with our use of [`table`](https://www.npmjs.com/package/table) within the [string formatter](https://github.com/stylelint/stylelint/blob/master/src/formatters/stringFormatter.js). Possibly when [we calculate the width of the columns](https://github.com/stylelint/stylelint/blob/master/src/formatters/stringFormatter.js#L79-L86).

I'm going to label this up as "need help". Help tracking down and fixing this issue would be greatly appreciated.

**Comment by davidtheclark** [comment]
Can you setup a reproducible test case for this?

**Comment by evilebottnawi** [comment]
@davidtheclark not reproducible, i think it is `window` problem, need testing on `window` machine

**Comment by woble** [comment]
Tried reproducing on Ubuntu (VMWare).

First run through CLI after installing stylelint:

```shell
foo.css
 2:42  ✖  Expected single quotes around font-family     font-family-name-quotes
          name "冬青黑体简体中文"
```

Any run after that results in script error:

```shell
Error: Subject parameter value width cannot be greater than the container width.
    at exports.default (/usr/lib/node_modules/stylelint/node_modules/table/dist/alignString.js:100:15)
    at /usr/lib/node_modules/stylelint/node_modules/table/dist/alignTableData.js:37:50
    at arrayMap (/usr/lib/node_modules/stylelint/node_modules/lodash/_arrayMap.js:16:21)
    at map (/usr/lib/node_modules/stylelint/node_modules/lodash/map.js:51:10)
    at /usr/lib/node_modules/stylelint/node_modules/table/dist/alignTableData.js:29:34
    at arrayMap (/usr/lib/node_modules/stylelint/node_modules/lodash/_arrayMap.js:16:21)
    at map (/usr/lib/node_modules/stylelint/node_modules/lodash/map.js:51:10)
    at exports.default (/usr/lib/node_modules/stylelint/node_modules/table/dist/alignTableData.js:28:30)
    at exports.default (/usr/lib/node_modules/stylelint/node_modules/table/dist/table.js:131:39)
    at formatter (/usr/lib/node_modules/stylelint/dist/formatters/stringFormatter.js:157:33)
```

**Comment by davidtheclark** [comment]
Can we write a *test* that fails, I mean?

**Comment by evilebottnawi** [comment]
@woble can you get information about old versions (ubuntu, stylelint, nodejs, npm and etc)?

**Comment by woble** [comment]
@evilebottnawi Didn't think of writing it down. I tried it again with fresh Ubuntu 15.10, node 4.4.4, npm 2.15.1 as well as 3.8.9 and everything worked fine. So I guess, as stated earlier, it's very much related to Windows only.

**Comment by evilebottnawi** [comment]
@davidtheclark reproduce error before merge, after merge all OK,
but get strange number
```shell
test.css
 2:3  ✖  Expected single quotes around font-family name "冬青黑体简体中文"   font-family-name-quotes
   4 <-- strange number
```

**Comment by evilebottnawi** [comment]
@davidtheclark it is `2:34`

**Comment by davidtheclark** [comment]
That must be a bug with the way the table is being constructed, right? — maybe string-width is not as accurate as it needs to be? Or something?

**Comment by evilebottnawi** [comment]
@davidtheclark Right. Now I'm trying to figure it out, but this bug is now out, can be closed
